tree/treeplayer/inc/ROOT/TDFInterface.hxx

```diff
@@ -395,7 +397,7 @@ public:
       };
       snapCall << ">(\"" << treename << "\", \"" << filename << "\", "
                << "*reinterpret_cast<std::vector<std::string>*>(" // vector<string> should be ColumnNames_t
-               << &columnList << "));";
+               << &columnList << ")," << "static_cast<const SnapshotOptions&>(" << &options << "));";
```

Comment:
casting an address to a reference, might be a typo

Reply:
Ah, it certainly is! I think my last amend didn't work :-) Now I know why the tests failed.